crypto/include/auth.h

```diff
@@ -156,16 +156,6 @@ srtp_err_status_t srtp_auth_type_self_test(const srtp_auth_type_t *at);
 srtp_err_status_t srtp_auth_type_test(const srtp_auth_type_t *at,
                                       const srtp_auth_test_case_t *test_data);
 
-/*
- * srtp_replace_auth_type(ct, id)
- *
- * replaces srtp's kernel's auth type implementation for the auth_type id
- * with a new one passed in externally.  The new auth type must pass all the
- * existing auth_type's self tests as well as its own.
- */
-srtp_err_status_t srtp_replace_auth_type(const srtp_auth_type_t *ct,
-                                         srtp_auth_type_id_t id);
-
```

Comment:
The `replace` functions are API, not dead code - they're to allow a third-party crypto implementation to be injected.

Reply:
That's a little surprising to me; I didn't realize the `crypto/include` headers were considered public API.

Reply:
Just `crypto/include/cipher.h`, `crypto/include/auth.h`, and `crypto/include/crypto_types.h` are public - see the `install` targets in `Makefile.in` or `CMakeLists.txt`.